sbin/build.sh

```diff
@@ -760,10 +768,11 @@ addSBOMMetadataPropertyFromFile() {
   local jsonFile="${3}"
   local name="${4}"
   local propFile="${5}"
+  value="N.A"
```

Comment:
Should this be decalred local as well?  I forget my bash syntax...

Reply:
dont have to be in the function. 
but to avoid complication, i can set it to be a local variable.